README.md

```diff
@@ -7,23 +7,26 @@ The resources/services/activations/deletions that this module will create/trigge
 - One `google_dns_managed_zone` for the zone
 - Zero or more `google_dns_record_set` for the zone records
 
+## Compatibility
+
+ This module is meant for use with Terraform 0.12. If you haven't [upgraded](https://www.terraform.io/upgrade-guides/0-12.html)
+  and need a Terraform 0.12.x-compatible version of this module, the last released version intended for
+  Terraform 0.11.x is [1.0.0](https://registry.terraform.io/modules/terraform-google-modules/cloud-dns/google/1.0.0).
+
 ## Usage
 
 Basic usage of this module for a private zone is as follows:
 
 ```hcl
-module "dns-zone-foo" {
-  source     = "terraform-google-modules/cloud-dns/google"
-  version    = "~> 0.1"
-  project_id = "my-project"
-  zone_type  = "private"
-  name       = "Foo zone."
-  dns_name   = "foo.local."
-  private_visibility_config = [{
-    networks = [{
-      network_url = "my-vpc"
-    }]
-  }]
+module "dns-private-zone" {
+  source     = "../.."
+  project_id = var.project_id
```

Comment:
I recommend *not* using variables in the README example, as they make the module intent less clear. Having hard-coded values is a better example of what each input actually expects.